Sources/Shapes.swift

```diff
@@ -201,17 +201,57 @@ public extension Path {
 }
 
 public extension Mesh {
+    /// A choice of the face directions that Euclid generates for polygons.
```

Comment:
@nicklockwood Not time critical, but when you get a chance - would you look over this addition and makes sure it lines up with how you'd like it presented? I read through the code a bit to (try and) verify my understanding of what it represented, but I'm not 100% confident here.

Reply:
This looks good to me.